src/components/Login/Login.js

```diff
@@ -11,12 +11,25 @@ import strings from '../../localization';
 import { login, actionTypes } from '../../actions/UserActions';
 import getUser from '../../selectors/UserSelector';
 import loadingSelector from '../../selectors/LoadingSelector';
-import styles from './styles';
-import { fetchData } from '../../actions/APIActions'
+import { fetchData } from '../../actions/APIActions';
 import { errorsSelector } from '../../selectors/ErrorSelector';
+import styles from './styles';
+import reciclandoLogo from './../../assets/images/Logo03.png';
+import avatar from './../../assets/ic_user/ic_user128.png';
```

Comment:
Maybe in the future we can add an index.js file for the assets folder and import everything in a single line.